client/lib/analytics/ad-tracking.js

```diff
@@ -580,7 +591,7 @@ export function isCurrentUserMaybeInGdprZone() {
  * @returns {void}
  */
 export function trackCustomFacebookConversionEvent( name, properties ) {
-	window.fbw && window.fbq( 'trackCustom', name, properties );
+	window.fbq && window.fbq( 'trackSingleCustom', TRACKING_IDS.facebookInit, name, properties );
```

Comment:
I searched the Calypso code and indeed fbw does not seem to be set, which means this method was never working. I see that the method is used in analytics/middleware.js when submitting data to multiple tracking services. After this is merged, we should observe if something strange happens to our Facebook reporting (since probably it hasn't been working for months).